src/app/modules/todo/components/done/done.component.html

```diff
@@ -17,7 +17,7 @@ <h2>Done</h2>
       (cdkDropListDropped)="drop($event)"
     >
       <div class="example-box" *ngFor="let item of doneList" cdkDrag>
-        {{ item }}
+        {{ item }} Hello Guys!! heyyyyy
```

Comment:
I think only one "y" looks better
```suggestion
        {{ item }} Hello Guys!! hey
```